conu/apidefs/image.py

```diff
@@ -170,6 +175,53 @@ def create_container(self, container_params):
         """
         raise NotImplementedError("create_container method is not implemented")
 
+    def run_in_pod(self, namespace="default"):
+        """
+        run image inside Kubernetes Pod
+        :param namespace: str, name of namespace where pod will be created
+        :return: Pod instance
+        """
+        config.load_kube_config()
+        api = client.CoreV1Api()
+
+        image_data = self.get_metadata()
+
+        # convert environment variables to Kubernetes objects
+        env_variables = []
+        for key, value in image_data.env_variables.items():
+            env_variables.append(client.V1EnvVar(name=key, value=value))
+
+        # convert exposed ports to Kubernetes objects
+        exposed_ports = []
+        if image_data.exposed_ports is not None:
+            for port in image_data.exposed_ports:
+                try:
+                    protocol = port.split("/", 1)[1]
+                    exposed_ports.append(client.V1ContainerPort(container_port=port.split("/", 1)[0],
+                                                                protocol=protocol))
+                except IndexError:  # protocol is not defined in image metadata
+                    exposed_ports.append(client.V1ContainerPort(container_port=port.split("/", 1)[0]))
+
+        # generate random container name
```

Comment:
would it be possible to make it not random? how about something like '{image-name}-{username}-{random-4-letters}'